blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -383,11 +384,14 @@ export const actions = {
                         // run. No need to create another i.e. ignore this event.
                         return;
                     }
+                    if (parseInt(run.id, 10) > nextId) { // need the estimated next id
+                        nextId = parseInt(run.id, 10);
+                    }
                 }
```

Comment:
yes how did this work in the past? I thought it used the queue id (cc @imeredith).